### PR TITLE
build: upgrade husky and lint staged to latest, remove unsed config files

### PR DESCRIPTION
**Pull request by ChristianMurphy**
Nota Bene: lint-staged now auto adds changes made during it's scripts to the git staging area/index.

### CODE REVIEW

File: lint-staged.config.js [hunk 1]
```diff
@@ -1,13 +0,0 @@
...
-const micromatch = require("micromatch");
-
-module.exports = {
-  "*.js": files => {
-    // from `files` filter those _NOT_ matching `*test.js`
-    const match = micromatch.not(
-      files,
-      "elements/**/*.*.js **/analysis-error.json themes/**/*.*.js libraries/**/*.js"
-    );
-    return `eslint ${match.join(" ")}`;
```
Comment (ChristianMurphy) [inline lint-staged.config.js:11]: this appears to be left over?
ESLint doesn't appear to have a configuration in this project

File: package.json [hunk 2]
```diff
@@ -127,10 +127,7 @@
     },
     "lint-staged": {
         "*.{js,json}": "prettier --write",
-        "*.{png,jpeg,jpg,gif,svg}": [
-            "imagemin-lint-staged",
-            "git add"
```
Comment (ChristianMurphy) [inline package.json:132]: files are now auto added after lint staged scripts run

**Review by btopro — APPROVED** [review]

**Review by btopro — APPROVED** [review]

**Comment by btopro** [comment]
odd it's finding a merge conflict here, arne't you just removing that file?

**Comment by btopro** [comment]
removed the file via my commit and merge went through

**Pull request merged**